### DISCUSSION TITLE
Seeking Collaborators for a Modern Whisper GUI Project

### DISCUSSION THREAD

**Discussion by rudymohammadbali** [openai/whisper#1692] [General]
### Hello,

I am looking for a talented individual with expertise in customtkinter to collaborate on the development of a modern Whisper GUI. This project will be open-source, and I am eager to assemble a team of passionate contributors. If you are interested in joining this effort, please share your Discord user ID with me.

I have already designed the final UI using Figma and have also implemented some initial functions for transcribing audio files. Your expertise in customtkinter will be invaluable in bringing this project to life.

Thank you for your consideration, and I look forward to working together on this exciting project!

**Comment by rudymohammadbali** [comment]
![Whisper GUI](https://github.com/openai/whisper/assets/63475761/1b93566f-c575-499b-bd9c-51d377ad4909)

**Comment by AIhasArrived** [comment]
Hello , i was wondering if anyone has made an UI for this. What langage did you use to make it?

**Reply by rudymohammadbali** [reply]
Yes, I have already made gui application:

https://github.com/iamironman0/OpenAI-Whisper-GUI

But now I'm working on a new modern ui with more tools. It will release soon.
